src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreCommand.cs

```diff
@@ -302,6 +302,19 @@ private string ConcatAsString<T>(IEnumerable<T> enumerable)
             return result;
         }
 
+        /// <summary>
+        /// Accounts for using the restore commands on 2 projects living in the same path
+        /// </summary>
+        private bool VerifyCacheMatchesAssetsFile()
```

Comment:
nit: can you change the name to something more appropriate? like `VerifyProjectName` or `CompareProjectName`